commit 61a90fa006a6baef05dc1eada994b18e2a921d75 (refs/changes/96/896/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-11-14T11:57:29-08:00 (2 years, 11 months ago)
    
    TOOLS-1610 sdcadm's appending "-$buildstamp" to the version was naive and broke 'sdcadm self-update --latest' (workaround)

diff --git a/CHANGES.md b/CHANGES.md
index 31f41ac..b1da15d 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,11 @@
 
 # sdcadm Changelog
 
+## 1.13.0
+
+- TOOLS-1610 Bump ver as a workaround to get 'sdcadm self-update --latest'
+  working.
+
 ## 1.12.0
 
 - TOOLS-1499: add -C channel command line option to sdcadm experimental
diff --git a/lib/sdcadm.js b/lib/sdcadm.js
index 4b81de5..f61f90d 100644
--- a/lib/sdcadm.js
+++ b/lib/sdcadm.js
@@ -4069,8 +4069,8 @@ SdcAdm.prototype._selfAvailable = function _selfAvailable(cb) {
                 // Filter out versions before the current.
                 candidates = candidates.filter(function dropLowerVer(c) {
                     if (semver.lt(c.version, currVer)) {
-                        //log.trace({candidate: c, currVer: currVer},
-                        //    'drop sdcadm candidate (lower ver)');
+                        log.trace({candidate: c, currVer: currVer},
+                            'drop sdcadm candidate (lower ver)');
                         return false;
                     }
                     return true;
diff --git a/package.json b/package.json
index e23adab..d7bacbf 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdcadm",
   "description": "Administer a SmartDataCenter (SDC) standup",
-  "version": "1.12.0",
+  "version": "1.13.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
